<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="28APO2678" name="Safari Test" tests="1" failures="1" timestamp="17 Jun 2016 11:13:17 GMT" time="0.047" errors="0">
  <testcase name="@AfterTest tearDown" time="0.0" classname="compageObjectsTestsaviaDigital.Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at compageObjectsTestsaviaDigital.Login.tearDown(Login.java:106)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 14 stack frames]]>
    </failure>
  </testcase> <!-- @AfterTest tearDown -->
  <testcase name="test_login_Correct" time="0.0" classname="compageObjectsTestsaviaDigital.Login">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at compageObjectsTestsaviaDigital.Login.test_login_Correct(Login.java:42)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- test_login_Correct -->
</testsuite> <!-- Safari Test -->
